src/main/resources/META-INF/plugin.xml

```diff
@@ -113,21 +119,17 @@
 
             <separator/>
 
-            <action id="GitMachete.RebaseCurrentBranchOntoParentAction"
-                    class="com.virtuslab.gitmachete.frontend.actions.toolbar.RebaseCurrentBranchOntoParentAction"
-                    icon="MacheteIcons.REBASE">
-                <override-text place="GitMacheteToolbar"/>
-            </action>
-
             <action id="GitMachete.FetchAllRemotesAction"
                     class="com.virtuslab.gitmachete.frontend.actions.toolbar.FetchAllRemotesAction"
                     icon="MacheteIcons.FETCH">
                 <override-text place="GitMacheteToolbar"/>
             </action>
 
-            <action id="GitMachete.PullCurrentBranchAction"
-                    class="com.virtuslab.gitmachete.frontend.actions.toolbar.PullCurrentBranchAction"
-                    icon="MacheteIcons.PULL">
+            <separator/>
+
+            <action id="GitMachete.RebaseCurrentBranchOntoParentAction"
```

Comment:
Hmm now fetch and push actions are separated by rebase, which looks a bit odd:
![image](https://user-images.githubusercontent.com/3383210/86838901-038c1080-c0a1-11ea-8a81-dc05e9291e5b.png)

Reply:
Actually... now that use a snapshot version built from this PR, I think this single button could change between reset/pull/push/nothing, depending on the state of the branch (but this would break our rule that we don't hide actions from the toolbar, just disable them) :thinking:

Reply:
wow great idea :rocket: but the statement about hiding-disabling is true too :confused: 

We could implement features like this and hide them behind some registry key - feature flipper :rofl: 

Anyway, I would rather leave it for another issue & PR

Reply:
> Hmm now fetch and push actions are separated by rebase, which looks a bit odd:
> ![image](https://user-images.githubusercontent.com/3383210/86838901-038c1080-c0a1-11ea-8a81-dc05e9291e5b.png)

it looks like they were pointing the rebase :rofl:

Reply:
#427 created